department-of-veterans-affairs/va.gov-cms
# VACMS-10113: Changed label to span using a patch.

## Description

Closes #10113 

## Testing done
Tested all content moderation states to confirm that <label> has been replaced with <div>

## Screenshots


## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As user _uid_ with _user_role_
1. Go to a content page and look at "Current state" in the Editorial Workflow section at the bottom of the page.
2. Right click and "Inspect" the console in the browser window. The html tag around "Current state" should be a <div>.
3. Change the content state to the other options - draft, in review, etc. Verify that the label is still a <div>.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
